src/main/java/org/opentripplanner/openstreetmap/wayproperty/specifier/OsmSpecifier.java

```diff
@@ -10,21 +10,25 @@
  * How the scoring logic is implemented is the responsibility of the implementations.
  */
 public interface OsmSpecifier {
-  static Condition.Equals[] parseEqualsTests(String spec, String separator) {
+  static Condition[] parseEqualsTests(String spec, String separator) {
```

Comment:
I introduced this method name and it's bad. Can you please rename it to something like `parseConditions`?

Reply:
The name was good enough as long as it parsed only «equals» conditions. I forgot to change the name.